stockfish/models.py

```diff
@@ -91,30 +102,24 @@ def get_parameters(self) -> dict:
                unknowingly getting bugs. It has been replaced with 'get_engine_parameters()'."""
         )
 
-    def get_engine_parameters(self) -> dict:
-        """Returns the current engine parameters being used.
-
-        Returns:
-            Dictionary of current Stockfish engine's parameters.
-        """
-
-        return self._parameters
-
-    def update_engine_parameters(self, new_param_valuesP: Optional[dict]) -> None:
-        """Updates the stockfish parameters.
+    def update_engine_parameters(self, parameters: Optional[dict]) -> None:
```

Comment:
Have updated the `parameters` argument, as the name `new_param_valuesP` is quite weird, and there's a mismatch between argument and the documentation a few lines below.

Reply:
:+1: